Packages/Tracking/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Preview package version dependency mismatch for XRI when using InputSystem 1.4.4
+- Control panel IR image / hand vertical alignment across device types and individual devices, through changing the handdling of the distortion matrix
```

Comment:
Do we need to mention the control panel here? the changelog only really relates to the Plugin, so we could just mention the passthrough distortion matrix, but not the control panel?

Reply:
Good suggestion, changed.